package.json

```diff
@@ -1,41 +1,60 @@
 {
-  "name": "snabbdom-looks-like",
```

Comment:
Please do not reformat the whole file, makes it really hard to see the actual changes

Reply:
I'm sorry, I forgot that GitHub can't ignore blank differences as GitLab...